kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -130,20 +130,6 @@ private PartitionedTableId getRecordTable(SinkRecord record) {
     return builder.build();
   }
 
-  private String getRowId(SinkRecord record) {
-    return String.format("%s-%d-%d",
-                         record.topic(),
-                         record.kafkaPartition(),
-                         record.kafkaOffset());
-  }
-
-  private RowToInsert getRecordRow(SinkRecord record) {
-    return RowToInsert.of(
-      getRowId(record),
-      recordConverter.convertRecord(record)
-    );
-  }
-
   @Override
   public void put(Collection<SinkRecord> records) {
 
```

Comment:
it's not letting me comment on the right line, but Map<PartitionTableId, TableWriter.Builder> should be changed to Map<PartitionTableId, TableWriterBuilder>